lib/bosh/workspace/project_deployment.rb

```diff
@@ -28,7 +28,15 @@ def merged_file
     end
 
     def manifest
-      @manifest ||= Psych.load(ERB.new(File.read(file)).result)
+      return @manifest unless @manifest.nil?
+      renderer = Bosh::Template::Renderer.new(context: stub.to_json)
+      @manifest = Psych.load(renderer.render(file))
+    end
+
+    def stub
+      return @stub unless @stub.nil?
+      stub_file = File.expand_path(File.join(file_dirname, "../stubs", file_basename))
+      @stub = File.exist?(stub_file) ? Psych.load(File.read(stub_file)) : {}
```

Comment:
one last thing, could you add a unit test for this line. Since it is quite an important for backward compatibility that it does not fail when there is no stub file.